.evergreen/config.yml

```diff
@@ -1503,7 +1573,7 @@ tasks:
           TOPOLOGY: server
       - func: add aws auth variables to file
       - func: setup aws env
-      - *ref_0
+      - func: run aws auth test with regular aws credentials
```

Comment:
all these changes are a result of spreading the function instead of sharing the reference in the config.  I prefer not to have anchor tags in our config so I changed it, but I can revert.